### DISCUSSION TITLE
Next release of DEV.env

### DISCUSSION THREAD

**Discussion by luisaveiro** [luisaveiro/dev.env#1] [Ideas]
I developed DEV.env to help developers automate parts of the setup process when joining a new development team.

I plan to rewrite the Shell script, including new console commands, cleaner console output and improved developer experience using the DEV.env console.

Although I have my wishlist for the list of features that I want to include, I would like feedback from my community on how I can improve DEV.env.

Do you have a specific feature in mind? Please comment below with your suggestion and I'll try to include it in this release.

**Comment by ageddesi** [comment]
What about a /help command that could take you to the projects associated confluence/wiki page

**Reply by luisaveiro** [reply]
Awesome idea. DEV.env provides a `repositories-template.yml` file to install multiple repositories at once. I can include a `documentationUrl` field.
